src/backends/interflop-vprec/interflop_vprec.c

```diff
@@ -843,13 +843,11 @@ void _interflop_enter_function(interflop_function_stack_t *stack, void *context,
       function_inst->input_args[i].min_range = INT_MAX;
       function_inst->input_args[i].max_range = INT_MIN;
       function_inst->input_args[i].exponent_length =
-          (type == FDOUBLE || type == FDOUBLE_PTR)
-              ? VPREC_RANGE_BINARY64_DEFAULT
-              : VPREC_RANGE_BINARY32_DEFAULT;
+          (type == FDOUBLE || type == FDOUBLE_PTR) ? function_inst->OpsRange64
```

Comment:
clang-format?

Reply:
It seems this has passed the clang-format test in the builder bot though...